internal/imageserver/imageserver.go

```diff
@@ -54,7 +54,7 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// if anything fails remove the workdir, if create succeeds it will remove the workdir so this will be a noop
 	defer os.RemoveAll(isoWorkDir)
 
-	// TODO: carbonin improve this to wait for some timout (use ctx?) instead of erroring on a lock failure immediately
+	// TODO: openshift improve this to wait for some timout (use ctx?) instead of erroring on a lock failure immediately
```

Comment:
nit: I guess this replace isn't on purpose

Reply:
:man_facepalming: